packages/create-rise/template-react-navigation/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "template-react-navigation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "rise start",
+    "eject": "rise eject"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@rise-tools/server": "latest",
+    "@rise-tools/kitchen-sink": "latest"
```

Comment:
this is a totally non-blocking nitpick that we can fix in a follow-up, but just wanted to point out that:
1. This example should be renamed to `template-basic` or...
2. We should add `kit-react-navigation` as a dependency
Reason being that React Navigation is not included in the `kitchen-sink` as we wanted to make it routing library agnostic.